src/move.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-console */
+'use strict';
+
+const { existsSync } = require('fs');
+const fs = require('fs/promises');
+const path = require('path');
+
+const moveFile = async(fromPath, toPath) => {
+  if (!existsSync(fromPath)) {
+    console.error(`Source file "${fromPath}" does not exist!`);
+
+    return;
+  }
+
+  try {
+    let newPath = toPath;
+
+    if (existsSync(newPath)) {
+      newPath = path.join(toPath, '/', path.basename(fromPath));
+
+      await fs.rename(fromPath, newPath);
+    } else {
+      await fs.rename(fromPath, newPath);
+    }
```

Comment:
It can be shorten like this. Nothing chages in logic, but less code and easier to read 
```suggestion
    const newPath = existsSync(toPath)
      ? path.join(toPath, "/", path.basename(fromPath))
      : toPath;

    await fs.rename(fromPath, newPath);
```